### PR TITLE
chore: Lazy load paypal button

### PR DESCRIPTION
**Pull request by iamareebjamal**
Related: #3634 

Vendor JS reduced from 4.1 MB to 3 MB

Paypal Checkout lib itself was of 1 MB

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- ember-cli-build.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/fossasia/open-event-frontend/pullRequest?prid=4719233&bid=15758022)

File: ember-cli-build.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 const MergeTrees = require('broccoli-merge-trees');
 const Funnel = require('broccoli-funnel');
 const targets = require('./config/targets');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```
Comment (kushthedude) [inline ember-cli-build.js]: Why adding this dependency ? Also how it is related to reduce the bundle sizes of js chunks ?

Comment (iamareebjamal) [reply ember-cli-build.js]: All can be answered if you search the name of the dependency

Comment (kushthedude) [reply ember-cli-build.js]: 👍

**Pull request merged**